src/modules/music/index.ts

```diff
@@ -5,9 +5,45 @@ import { ModuleBase } from '../../internal/module-base';
  *
  * ### Overview
  *
- * Generate a random music genre with [`genre()`](https://fakerjs.dev/api/music.html#genre) or song name with [`songName()`](https://fakerjs.dev/api/music.html#songname). Both may be localized.
+ * Generate random music content.
+ *
+ * For a random album name, use [`albumName()`](https://fakerjs.dev/api/music.html#albumname).
+ *
+ * For a random artist, use [`artist()`](https://fakerjs.dev/api/music.html#artist).
+ *
+ * For a random genre, use [`genre()`](https://fakerjs.dev/api/music.html#genre).
+ *
+ * For a random song name, [`songName()`](https://fakerjs.dev/api/music.html#songname).
+ *
+ * All data types may be localized.
  */
 export class MusicModule extends ModuleBase {
+  /**
+   * Returns a random album name.
+   *
+   * @example
+   * faker.music.albumName() // '1989'
+   *
+   * @since 8.5.0
```

Comment:
We will not be releasing a version 8.5 as we already work on bringing the current codebase to v9.0. So these methods will (most likely) land in v9.0 as well.

```suggestion
   * @since 9.0.0
```